### DISCUSSION TITLE
Please helping with CR Touch , Mesh Viewer Shows , Z Compensate dosen't

### DISCUSSION THREAD

**Discussion by Rockument** [Jyers/Marlin#1425] [Q&A]
I've install CR Touch and flash screen display , firmware to 1.3.5 after created a new mesh and set Z-offset probe save all settings. at mesh viewer show all the data points. In Cura added G29 on a GCODE. 

But,When it running first all home and try G29 level 25 points when i look at the Z position at bottom right corner it never changes or try to compensate a Z position when it print some corners are high somes are low.

How todo to fixing it i had try to 1.3.4 or even try a Creality firmware 1.1.2 and try to compile marlink 2.0.9 all no joy.

Thanks.

![21-09-24 18-54-26 7708](https://user-images.githubusercontent.com/55097385/134767066-0d92bd78-dcca-4e10-82cd-5d014b11409b.jpg)

![21-09-24 16-16-40 7699](https://user-images.githubusercontent.com/55097385/134767079-6da92987-830f-4332-8b45-01b551a6e518.jpg)

![21-09-24 17-21-08 7702](https://user-images.githubusercontent.com/55097385/134767098-0da4efdb-6268-43f9-9cf3-b4f37bc36270.jpg)

**Comment by Rockument** [comment]
Thanks will try.

**Reply by cmadon** [reply]
Did this fix your issue? What steps did you use?
